tutorials/stats-source-space/60_cluster_rmANOVA_spatiotemporal.py

```diff
@@ -285,9 +285,7 @@ def stat_fun(*args):
 
 inds_t, inds_v = [
     (clusters[cluster_ind]) for ii, cluster_ind in enumerate(good_cluster_inds)
-][
-    0
-]  # first cluster
```

Comment:
@drammock here's an example where things are reversed and are looking better with ruff